apps/prairielearn/src/migrations/20230531184500_assessment_access_policies.sql

```diff
@@ -0,0 +1,19 @@
+-- Create assessment_access_policies table
+CREATE TABLE IF NOT EXISTS
+  assessment_access_policies (
+    id BIGSERIAL PRIMARY KEY,
+    assessment_id BIGINT REFERENCES assessments (id) ON UPDATE CASCADE ON DELETE CASCADE NULLS NOT DISTINCT,
+    user_id BIGINT REFERENCES users (user_id) ON UPDATE CASCADE ON DELETE CASCADE NULLS NOT DISTINCT,
+    group_id BIGINT REFERENCES groups (id) ON UPDATE CASCADE ON DELETE CASCADE NULLS NOT DISTINCT,
+    start_date TIMESTAMP WITH TIME ZONE NOT NULL,
+    end_date TIMESTAMP WITH TIME ZONE NOT NULL,
+    credit INTEGER NOT NULL,
+    note TEXT,
+    created_by BIGINT REFERENCES users (user_id) NOT NULL,
```

Comment:
Conventionally a `NOT NULL` constraint should appear immediately after the column type.

```suggestion
    created_by BIGINT NOT NULL REFERENCES users (user_id),
```